### ISSUE TITLE
Feature to add please - Auto Allow

### ISSUE THREAD

**Issue by REZISTENCE** [andryou/scriptsafe#192]
Hi there, really like where your extension sits as a script block on Chrome
Would/Is it possible to get the options show in the attachment added to your blocker?

![auto-allow](https://cloud.githubusercontent.com/assets/26421284/23926749/f47484c4-0965-11e7-8222-f2eedb471957.jpg)

It would make it much more intuitive
Thank you for your time

**Comment by StepnSteph** [comment]
Temporarily allowing top level domains would be a great feature. Too many websites just straight up don't work without it so I end up manually enabling it all the time anyway.

**Comment by Usalabs** [comment]
I can temporarily allow top level domains, just by clicking on the 'scriptsafe' icon, then hit the temporary button, when on a www.domain.com then I use scriptsafe to temporarily allow subdomains and/or block them too so that if a web page calls another domain/sub-domain to display ads, I can block that, just by not setting anything in scriptsafe icon, but if a video calls another domain/subdomain to play that video, then I temporarily allow it, so that I can watch that video without any ads from the calling web page.

**Comment by StepnSteph** [comment]
Yeah. What I meant is that it needs to be automatic. So that if I visit Fark.com for example I don't have to always manually temporarily enable scripts for that site.

Far too many websites just flat-out don't work without scripts enabled so I'm always either babysitting the extension, or just turning it off completely. Neither are good results.

**Comment by simonmadsen** [comment]
I agree with this. There should be possibilty to allow all subdomains at once for a site, and not each of them individually.
